server/src/test/java/org/opensearch/common/settings/WriteableSettingTests.java

```diff
@@ -462,6 +463,7 @@ public void testVersionSetting() throws IOException {
         }
     }
 
+    @Ignore("https://github.com/opensearch-project/OpenSearch/issues/5504")
```

Comment:
This test of course fails as it uses setAccessible and also adds SuppressForbidden!? There was a reason why the forbiddenapis frbids setAccessible!

Reply:
🤷‍♂️

Reply:
I think you're saying that this is by design. Let's rewrite the test as such!

Reply:
See other PR, I made a proposal. Alternatively remove the test. Because if the fields are "final" theres no need to test any other value than the final value.